pkg/container/types/bytes.go

```diff
@@ -59,6 +59,8 @@ func (v *Varlena) SetOffsetLen(voff, vlen uint32) {
 	s[2] = vlen
 }
 
+// do not use this function, will be deleted in the future
+// use BuildVarlenaFromValena or BuildVarlenaFromByteSlice instead
```

Comment:
Just remove this function.

Reply:
used in ut test. have tried but hard to remove.
will delete this function in the future